tests/backup/backup_upgrade_test.go

```diff
@@ -759,3 +760,552 @@ var _ = Describe("{PXBackupEndToEndBackupAndRestoreWithUpgrade}", func() {
 		CleanupCloudSettingsAndClusters(backupLocationMap, cloudAccountName, cloudAccountUid, ctx)
 	})
 })
+
+var _ = Describe("{PXBackupOcpUpgradeTest}", func() {
```

Comment:
I do not see what makes this test case OCP-specific since adjusting flags and environment variables could allow it to run on other platforms too. What about using `PXBackupUpgradeClusterTest`?

Reply:
Once we plugin other cases this will make sense. Doesn't do any harm to change it.